File: AndroidBase_VClassify/app/src/main/java/io/inteliedoit/androidbase_vclassify/ui/main/MainActivity.java
package io.inteliedoit.androidbase_vclassify.ui.main;

import android.os.Bundle;
import android.support.annotation.NonNull;
import android.support.design.widget.BottomNavigationView;
import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentTransaction;
import android.support.v7.app.ActionBar;
import android.support.v7.app.AppCompatActivity;
import android.view.MenuItem;


import io.inteliedoit.androidbase_vclassify.R;
import io.inteliedoit.androidbase_vclassify.ui.main.fragment.SubjectListViewGradeFragment;
import io.inteliedoit.androidbase_vclassify.ui.main.fragment.UserSettingFragment;
import io.inteliedoit.androidbase_vclassify.ui.main.fragment.VoiceCommandFragment;
import io.inteliedoit.androidbase_vclassify.ui.main.fragment.user.SubjectListFragment;
import io.inteliedoit.androidbase_vclassify.ui.main.fragment.viewing.VGFirstYearFirtSemFragment;

public class MainActivity extends AppCompatActivity {

    private ActionBar toolbar;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        BottomNavigationView navigation = (BottomNavigationView) findViewById(R.id.navigation);
        navigation.setOnNavigationItemSelectedListener(mOnNavigationItemSelectedListener);

        toolbar = getSupportActionBar();

        // load the store fragment by default
        toolbar.setTitle("Schedule");
        loadFragment(new SubjectListViewGradeFragment());
    }

    private BottomNavigationView.OnNavigationItemSelectedListener mOnNavigationItemSelectedListener
            = new BottomNavigationView.OnNavigationItemSelectedListener() {

        @Override
        public boolean onNavigationItemSelected(@NonNull MenuItem item) {
            Fragment fragment;
            switch (item.getItemId()) {
                case R.id.navigation_schedule:
                    toolbar.setTitle("Subject");
                    fragment = new SubjectListViewGradeFragment();
                    loadFragment(fragment);
                    return true;
                case R.id.navigation_command:
                    toolbar.setTitle("Command");
                    fragment = new VoiceCommandFragment();
                    loadFragment(fragment);
                    return true;
                case R.id.navigation_setting:
                    toolbar.setTitle("Setting");
                    fragment = new UserSettingFragment();
                    loadFragment(fragment);
                    return true;
            }

            return false;
        }
    };

    private void loadFragment(Fragment fragment) {
        // load fragment
        FragmentTransaction transaction = getSupportFragmentManager().beginTransaction();
        transaction.replace(R.id.frame_container, fragment);
        transaction.addToBackStack(null);
        transaction.commit();
    }

}

File: app/src/main/java/io/inteliedoit/vclassify/ui/main/MainActivity.kt
package io.inteliedoit.vclassify.ui.main

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import io.inteliedoit.vclassify.R
import android.support.v4.view.ViewPager
import android.support.design.widget.TabLayout
import android.support.v4.app.FragmentPagerAdapter
import android.view.View


import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;

import android.support.v7.widget.Toolbar;

import java.util.ArrayList;


class MainActivity : AppCompatActivity() {


    private var toolbar: Toolbar? = null
    private var tabLayout: TabLayout? = null
    private var viewPager: ViewPager? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        toolbar = findViewById<View>(R.id.toolbar) as Toolbar
        setSupportActionBar(toolbar)

        supportActionBar!!.setDisplayHomeAsUpEnabled(true)

        viewPager = findViewById<View>(R.id.viewpager) as ViewPager?
        setupViewPager(viewPager)

        tabLayout = findViewById<View>(R.id.tabs) as TabLayout?
        tabLayout!!.setupWithViewPager(viewPager)
    }

    private fun setupViewPager(viewPager: ViewPager?) {
        val adapter = ViewPagerAdapter(supportFragmentManager)
        adapter.addFragment(ScheduleListFragment(), "Schedule")
        adapter.addFragment(VoiceCommandFragment(), "Profile")
        viewPager!!.adapter = adapter
    }

    internal inner class ViewPagerAdapter(manager: FragmentManager) : FragmentPagerAdapter(manager) {
        private val mFragmentList = ArrayList<Fragment>()
        private val mFragmentTitleList = ArrayList<String>()

        override fun getItem(position: Int): Fragment {
            return mFragmentList.get(position)
        }

        override fun getCount(): Int {
            return mFragmentList.size
        }

        fun addFragment(fragment: Fragment, title: String) {
            mFragmentList.add(fragment)
            mFragmentTitleList.add(title)
        }

        override fun getPageTitle(position: Int): CharSequence {
            return mFragmentTitleList.get(position)
        }
    }
}